specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3231,230 @@
       ],
       "description": "Parameter required for creating a linked server to redis cache."
     },
+    "RedisCacheAccessPolicy": {
```

Comment:
>RedisCacheAccessPolicy [](http://example.com/codeflow?start=5&length=22)

I'd follow the convention of calling this RedisCacheAccessPolicyProperties since it is properties of the resource